src/utils/read-json.ts

```diff
@@ -0,0 +1,12 @@
+import type * as d from '../declarations';
+
+/**
+ * Reads and parses a JSON file from the given `path`
+ * @param sys The system where the command is invoked
+ * @param path the path on the file system to read and parse
+ * @returns the parsed JSON
+ */
+export async function readJson(sys: d.CompilerSystem, path: string): Promise<any> {
+  const file = await sys.readFile(path);
+  return !!file && JSON.parse(file);
```

Comment:
Something to consider as we pull this out - our use case is more general. In other words, although we wrote this for telemetry, other consumers may need to change this in order to make it work for them, which can have an affect on the original (telemetry) use case.

An example of this is we don't handle when/if `JSON.parse` throws. We made some assumptions around this in the original use case, which may not hold going forward.

No action required ATM, but something to be cognizant of/think about as a semi-rhetorical question: "If someone were to change this to re-`throw` an error from `JSON.parse`, how would my tests catch it? How would telemetry be affected?